news/asciidoctorj-1-6-0-released.adoc

```diff
@@ -0,0 +1,118 @@
+
+Asciidoctor 1.6.0 is a transitional release in preparation for making the switch to Semantic Versioning (or SemVer, for short) in 2.0.0.
+SemVer is a logical system for choosing version numbers that uses a simple MAJOR.MINOR.PATCH sequence.
+A MAJOR version is for when you make incompatible changes.
+A MINOR version is for when you add functionality that's backwards compatible.
+A PATCH version is for when you make bug fixes (backwards compatible, of course).
+You can learn more at https://semver.org.
+Although it's not a perfect system, it should make it far easier to understand what a version number means than with the current system (which is really no system at all).
+
+Obviously, this release does not follow semantic versioning.
+However, given how long the 1.6.0 branch has been brewing, we wanted to allow it to land to gather feedback and tease out any unexpected surprises before making the transition to 2.0.0 and SemVer.
+If all goes well, we expect the 2.0.0 release to follow shortly after this one.
+At that point, we'll stick to versioning according to the SemVer rules.
+
+Switching to SemVer also means that, for the first time, AsciidoctorJ will be versioned independently of Asciidoctor.
+AsciidoctorJ has reached a point where the project is so much more than just the Asciidoctor RubyGem repackaged for the JVM.
+The project has a complete API, including Java-based extensions, and custom features that provide tight integration with the Java ecosystem (e.g., JUL, Ruby object bridging, etc).
+And that warrants its own version line.
+An independent version line will allow you to better understand what to expect with each release.
+More important, it will address the very situation that backed up the 1.6.0 release for several years.
+
+(You can still find out the underlying Asciidoctor release via the `Asciidoctor.asciidoctorVersion()` API).
+
+== New features
+
+Major API and packaging changes::
+Since the start of AsciidoctorJ, we've learned a whole lot about JRuby and how it bridges Ruby and Java objects.
+In order for AsciidoctorJ to continue moving forward, breaking changes had to be made to the API and packaging.
+Using the knowledge gained, we completely redesigned the integration to correct flaws in the original design.
+As a result, this redesign removes glitches you may have experienced when writing extensions or walking the document model with the 1.5.x releases.
+We also split the API and implementation classes into separate packages and JARs in order to isolate them.
+
+Reworked extension API::
+Thanks to the redesign, the JRuby based implementation is much better hidden.
+The interfaces for the Abstract Syntax Tree (AST) were renamed to better represent their purpose.
+For example, the AbstractNode interface was renamed to ContentNode, AbstractBlock was renamed to StructuralNode, and Inline was renamed to PhraseNode.
+And even more of the AST is now mapped in Java, including list item and table cell nodes.
```

Comment:
Each sentence on lines 68 and 69 both start with "And". Maybe have one of them start differently, just to break up the rhythm? For example: Plus, even more of the AST ...

But this is just my personal preference, so feel free to ignore this comment :smile: